zokrates_core/src/typed_absy/mod.rs

```diff
@@ -298,6 +298,7 @@ pub enum BooleanExpression<T: Field> {
     Eq(Box<FieldElementExpression<T>>, Box<FieldElementExpression<T>>),
     Ge(Box<FieldElementExpression<T>>, Box<FieldElementExpression<T>>),
     Gt(Box<FieldElementExpression<T>>, Box<FieldElementExpression<T>>),
+    AndAnd(Box<BooleanExpression<T>>, Box<BooleanExpression<T>>),
```

Comment:
Looks great so far! Just wondering if we could call this `And` at this point as we're referring to the boolean And operation and not the `&&` character sequence.

Reply:
That's fair, I chose the name AndAnd to distinguish boolean and (&&) from bitwise and (&), which might implemented in the future. Maybe BoolAnd and BitAnd would be clearer.

Reply:
Oh I see, however in this context isn't it already clear we're talking about booleans as we're defining BooleanExpressions?

Reply:
I agree with renaming AndAnd here to just And - the BooleanExpression parameters are enough to distinguish it, and Eq follows the same naming convention, where the token is lexed as EqEq but written as Eq in this enum.

Reply:
Made the change - andand is now and